pokemongo_bot/step_walker.py

```diff
@@ -19,17 +19,17 @@ def __init__(self, bot, dest_lat, dest_lng):
             dest_lat,
             dest_lng
         )
-
-        self.speed = self.bot.config.walk_max - random() * (self.bot.config.walk_max - self.bot.config.walk_min)
+            
+        self.speed = uniform(self.bot.config.walk_min, self.bot.config.walk_max)
 
         self.destLat = dest_lat
         self.destLng = dest_lng
         self.totalDist = max(1, self.dist)
 
         if self.speed == 0:
-            self.steps = 1
-        else:
-            self.steps = (self.dist + 0.0) / (self.speed + 0.0)
+            self.speed = uniform(3,5)
```

Comment:
I don’t like that. Maybe someone wants to set ```walk_min``` and ```walk_max``` to 0. Maybe today it makes no sense, but it could be useful later on.
It’s better to set a default value to ```walk_min``` and ```walk_max```  when retrieving them from the config file, if they are not in it. 
Kinda like 
```
        self.walk_min = self.config.get("walk_min", 3)
        self.walk_max = self.config.get("walk_max", 5)

```